extensions/amazon-lambda/common-runtime/src/main/java/io/quarkus/amazon/lambda/runtime/AbstractLambdaPollLoop.java

```diff
@@ -125,6 +125,7 @@ public void run() {
                                 URL url = AmazonLambdaApi.invocationResponse(baseUrl, requestId);
                                 if (isStream()) {
                                     HttpURLConnection responseConnection = responseStream(url);
+                                    responseConnection.setRequestProperty("Content-Type", "application/json");
```

Comment:
Maybe. I was unsure about this after you provided me further insights about the purpose of this poll loop. The reason why I added this is because RestAssured does not work properly without this option. Because it receives or assumes (not sure what exactly...) "application/octet-stream", which cannot be deserialized to a specific model.

E.g. This was not possible to do:
```java
        var respBody = response.then().statusCode(200)
                .extract().body().as(BatchItemFailures.class);
```

extract().body().as complained because of the unexpected content type.

So with the new information I probably need to revert this change and in the tests I would need to deserialize the content manually. E.g. via Jackson.

Reply:
Although, Isn't it a bit strange, because for the incoming data funqy forces json right? Because jackson is used for the deserialization. Or am I missing something?

Reply:
Yes, funqy requires JSon, but that code works is for regular Quarkus integration with Lambda (quarkus-lambda extension)  What I'm saying is, don't set the content type if isStream() == true.

Reply:
I'll explain more.  You will break the quarkus-amazon-lambda extension if you add set the content type when isTream() == true

Reply:
It is removed now.